README.md

```diff
@@ -115,19 +115,26 @@ directory the variables from `.env` will be loaded into the environment.
 E.g. `foreman run rails server` or `foreman run rails console`.
 
 ## Quick Start 
-###Beginner Friendly Tips for New Contributors
+
+### Beginner Friendly Tips for New Contributors
+
 - After forking the repo, follow the steps described above under 'Bootstrap'. Mailtrap is optional.
 - (Install and) connect to Postgres server 
 - With everything properly installed, open the browser in development environment
 - The app should be available, with the database loaded with fake data.
 - To access all the functionality of the teams app, add yourself as an organizer.
     * In the browser: log in with your github account 
-    * In Rails Console:
-    ``` user = User.last``` #or ```user = User.find_by(github_handle: "yourgithubhandle") ```
-    ``` user.roles.create(name: "organizer") ```
-    You can assign yourself other roles in the same way. If however you assign
-    yourself a student role AND another role, that may lead to unexpected behavior in the app. In that case, remove the student role. 
-    - Refresh the browser to effectuate. You should see links for organizers. 
+    * In Rails Console:   
+      ```
+      user = User.last
+      user.roles.create(name: "organizer")
+      ```
+      Instead of using the last created User, you can also search for a
+      specific User with ```user = User.find_by(github_handle: "yourgithubhandle")```.
+
+      You can assign yourself other roles in the same way. If however you assign
```

Comment:
'the same way' is not so clear, with the sequence changed. Maybe something as : the same way as creating the organizer role.